libs/api/__tests__/api/private/index.ts

```diff
@@ -2156,7 +2156,7 @@ export type QueryRemotePeerProfileArgs = {
 
 
 export type QuerySettingArgs = {
-  name?: InputMaybe<Scalars['String']>;
+  name: Scalars['String'];
```

Comment:
Used to be optional in graphql and then verified to be non optional in the resolver. Changed it to be non optional by graphql